src/utils/build.js

```diff
@@ -178,7 +178,12 @@ const forceIncludeDumbPath = (appConfig, filePath) => {
   return (
     filePath.endsWith('package.json') ||
     filePath.endsWith('definition.json') ||
-    filePath.endsWith(`/bin/linux-x64-node-${nodeMajorVersion}/deasync.node`) || // Special, for zapier-platform-legacy-scripting-runner
+    filePath.endsWith(
+      // Special, for zapier-platform-legacy-scripting-runner
+      path.sep === '\\'
```

Comment:
You can change this whole ternary to use `path.join`, which takes care of detecting and using the separator correctly. Same goes for above, and basically any other use of `path.sep`.

```js
filePath.endsWith(path.join('bin', `linux-x64-node-${nodeMajorVersion}`, 'deasync.node'))
```